ironic/tests/unit/drivers/modules/test_agent.py

```diff
@@ -1662,7 +1662,11 @@ def test_get_clean_steps(self, get_steps_mock):
             {'step': 'create_configuration', 'interface': 'raid',
              'priority': 1},
             {'step': 'delete_configuration', 'interface': 'raid',
-             'priority': 2}]
+             'priority': 2},
```

Comment:
If we remove the changes to agent.py, then these test cases can be removed.

Reply:
I have removed this test case, as we don't makes any changes in agent.py.

Reply:
Can you back out the changes to this file?  It's still showing up as modified.

Reply:
Done